kbase-extension/static/kbase/css/kbaseNarrative.css

```diff
@@ -3125,6 +3119,70 @@ button.kb-data-obj {
 .kb-import-content .upload-options button:disabled {
     background-color: #FAFAFA;
     color: #929292;
+
+}
+
+/* style for Import As dropdown container */
+.kb-import-content .select2 .select2-selection--single {
```

Comment:
Question: do you know if it is possible to give the elements that select2 creates (`span`? `div`? I can't check because kbase is down 😖 ) specific classes, e.g. `.kb-import-select2-container`, `.kb-import-select2-selection`, etc.?

Reply:
i do know you can add a class to the main dropdown/<span> container element that gets created (.select2 as referenced here), not sure about further tagging classes down the tree.

Reply:
I was just thinking about trying to adopt a more modular/BEM-type approach to css, where components are given a specific class and styled appropriately, rather than relying on the cascade--and thus tying us into a certain DOM structure--so much. E.g. in this case, the `.kb-import-content` class is on a container div several levels up the DOM tree; instead of doing that, give the select2 container element a specific class (e.g. `.kb-import-select2__container`).

I'm already seeing css problems from relying on a massively nested structure in the CSS => SCSS ticket so anything that can be done to avoid making more such problems would be great!

Reply:
i hear you and agree. i was not so happy with these convoluted references - especially since select2 has a very generic naming scheme it makes it hard to find context when you are looking through the code. 

i will see about how i can add some more helpful and structured classes for the dropdown component

Reply:
just pushed a change with some new classes - we still unfortunately are a bit tied to the select2 dom structure to trigger changes when the dropdown is opened vs not (for some reason plain ol :focus and :active alone don't quite do it). let me know what you think, open to further feedback on how to improve